<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team Fortress 2 bot kicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="4">
     <widget class="QCheckBox" name="reminders">
      <property name="text">
       <string>Repeatedly send chat messages about joined bots</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QCheckBox" name="messageonjoin">
      <property name="text">
       <string>Send a chat message when a bot joins</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="4">
     <widget class="QTextBrowser" name="consoleLog"/>
    </item>
    <item row="8" column="3">
     <widget class="QSpinBox" name="spinBox">
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="TimeoutLabel">
      <property name="text">
       <string>Timeout between trying to kick bots:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QCheckBox" name="autokick">
      <property name="text">
       <string>Automatically vote kick bots in your team</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="startbutton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="steamUUID"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="TF2DirectoryLabel">
      <property name="text">
       <string>Team Fortress 2 Installation Directory:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="directory"/>
    </item>
    <item row="3" column="3">
     <widget class="QToolButton" name="selectDirectory">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="folder">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="SteamUUIDLabel">
      <property name="text">
       <string>Your Steam UUID (Optional. Used to know if bots are in your team):</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Based on &lt;a href=&quot;https://github.com/Googe14/tf2_bot_kicker&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;tf2_bot_kicker by Googe14 &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
